gridstatus/tests/source_specific/test_nyiso.py

```diff
@@ -545,6 +545,81 @@ def test_zonal_load_forecast_historical_date_range(self):
             (df["Interval End"] - df["Interval Start"]) == pd.Timedelta(minutes=60)
         ).all()
 
+    """get_interface_limits_and_flows_5_min"""
+
+    @pytest.mark.integration
+    @api_vcr.use_cassette(
+        "test_get_interface_limits_and_flows_5_min_historical_date_range.yaml",
+    )
+    def test_get_interface_limits_and_flows_5_min_historical_date_range(self):
+        start = self.local_start_of_today() - pd.DateOffset(days=10)
+        end = start + pd.Timedelta(days=1)
```

Comment:
I think we're still figuring out our conventions a bit, so open to doing it a different way. But I think when we implement VCR on a test I think we 

1. can remove the integration `mark`. 
2. `pytest.parameterize` the date args if possible (`start_of_local_today()` has proved mildly tricky/not possible and I do what you do here in that case, but maybe a more sustainable fix is available down the road)
3. use the context manager (vs the decorator) so we can dynamically name the fixture and do potentially some other things I'm forgetting at the moment. In this case in particular, it doesn't break like it used to with RECORD_MODE="once", but with RECORD_MODE="new_episodes" this is silently rerecording vs explicitly creating a new file (and I know I definitely prefer the latter).

```suggestion
    def test_get_interface_limits_and_flows_5_min_historical_date_range(self):
        start = self.local_start_of_today() - pd.DateOffset(days=10)
        end = start + pd.Timedelta(days=1)
        with nyiso_vcr.use_cassette(f"test_get_interface_limits_and_flows_5_min_historical_date_range_{date.strftime('%Y-%m-%d')}_{end.strftime('%Y-%m-%d')}"
        # test logic
```

Reply:
Okay, I've implemented 1 and 3 but I don't think 2 is relevant in this case.